packages/core-components/src/components/category/CategoryItem.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+
+export type ItemProps = {
+  icon?: React.ReactNode;
+  link?: string;
+  children: string;
+  boldTitle?: boolean;
+  small?: boolean;
+  handleClick?: () => void;
+  handleMouseEnter?: () => void;
+  handleMouseLeave?: () => void;
+};
+
+const CategoryItem: React.FC<ItemProps> = ({
+  icon,
+  link,
+  children,
+  boldTitle,
+  small,
```

Comment:
what is this `small` prop. Not sure how can I understand this

Reply:
same question. Have you fixed this!? why do you mark this as resolved?!

Reply:
I get what is `small` prop now. But still, why do we have to have this condition? Create another component and call it a day